omnibus-arbitrum-sepolia.toml

```diff
@@ -163,13 +163,15 @@ description = "https://pyth.network/developers/price-feed-ids#pyth-evm-mainnet"
 
 [provision.dai_mock_collateral]
 source = "mintable-token:1.8"
+target = "synthetix-mock-tokens:1.8@dai"
```

Comment:
Is this something new? Like a new cannon feature we spoke about before? Would be nice to have some context/links in the PR description.

Reply:
The main question is here - what effect are we expecting? What will happen when we publish? I can see from the build that there are no new contracts deployed, so my guess is that we would publish multple new packages on cannon publish?

Reply:
Sorry, just saw this. The idea here is that if/when this omnibus package is published, the subpackage would get published as `synthetix-mock-tokens:1.8@dai` instead of `mintable-token:1.8`. Renaming this operation to `clone` from `provision`, the idea is we "clone" the mintable-token package into a new synthetix-mock-tokens package.